### PR TITLE
Tc dev 2021 1

### PR DESCRIPTION
**Pull request by narrucmot**
Devicetree changes for axi_intc settings and some other cleanup.

### CODE REVIEW

File: recipes-bsp/device-tree/files/uz3eg-pciec/system-bsp.dtsi [hunk 1]
```diff
@@ -1,186 +1,199 @@
...
+         <24 4>, <25 4>, <26 4>, <27 4>,
+         <28 4>, <29 4>, <30 4>, <31 4>;
+   };
 };
 
 &psgtr {
-	/* sata, usb3, dp */
-	clocks = <&gtr_clk1>, <&gtr_clk2>, <&gtr_clk3>;
-	clock-names = "ref1", "ref2", "ref3";
+   /* NC, SATA, USB3, DP */
```
Comment (wpatterson-witekio) [inline recipes-bsp/device-tree/files/uz3eg-pciec/system-bsp.dtsi:178]: Why is NC being added to the comment but there is not an additional clock added? i.e. why does the comment list 4 descriptions for 3 clocks? In other `psgtr` nodes there are 4 clocks, maybe the comment just got copy/pasted from one of those?

Comment (narrucmot) [reply recipes-bsp/device-tree/files/uz3eg-pciec/system-bsp.dtsi:178]: The NC notes there is a No Connect.  There can be up to 4 GTR reference clocks described in the devicetree, but our hw platform only uses 3 and we note the 4th as NC.  This is what was recommended by the engineer at Xilinx.  The UZ7EV uses all 4 PS GTR reference clocks (though not connected the same to the Zynq MPSoC, and the 4th ref clock is used for PCIe).

Comment (wpatterson-witekio) [reply recipes-bsp/device-tree/files/uz3eg-pciec/system-bsp.dtsi:178]: Ah, that makes sense

**Review by wpatterson-witekio — APPROVED** [review]

**Review by tnizan-witekio — APPROVED** [review]

**Comment by AlbertaBeef** [comment]
Approved, with fix for axi_intc device-tree definition ( b2e7f6b )
Verified with vitis build of dpu design for u96v2-sbc-base platform.

**Pull request merged**